src/psyclone/tests/psyGen_test.py

```diff
@@ -838,16 +838,23 @@ def test_ompdo_directive_class_view(capsys):
             case["current_class"].view(omp_parallel_loop)
 
             out, _ = capsys.readouterr()
+
+            directive = colored("Directive", SCHEDULE_COLOUR_MAP["Directive"])
+            literal = colored("Literal", SCHEDULE_COLOUR_MAP["Literal"])
+            loop = colored("Loop", SCHEDULE_COLOUR_MAP["Loop"])
+            sched = colored("Schedule", SCHEDULE_COLOUR_MAP["Schedule"])
+            kern = colored("CodedKern", SCHEDULE_COLOUR_MAP["CodedKern"])
+
             expected_output = (
-                colored("Directive", SCHEDULE_COLOUR_MAP["Directive"]) +
-                case["current_string"] + "\n"
-                "    "+colored("Loop", SCHEDULE_COLOUR_MAP["Loop"]) +
-                "[type='',field_space='w1',it_space='cells', "
-                "upper_bound='ncells']\n"
-                "        "+colored("CodedKern",
-                                   SCHEDULE_COLOUR_MAP["CodedKern"]) +
-                " testkern_code(a,f1,f2,m1,m2) "
-                "[module_inline=False]")
+                directive + "[OMP parallel do]\n"
+                "    " + loop + "[type='', field_space='w1', it_space='cells',"
+                " upper_bound='ncells']\n"
+                "        " + literal + "[value:'NOT_INITIALISED']\n"
+                "        " + literal + "[value:'NOT_INITIALISED']\n"
+                "        " + literal + "[value:'1']\n"
+                "        " + sched + "[]\n"
+                "            " + kern + " testkern_code(a,f1,f2,m1,m2) "
+                "[module_inline=False]\n")
```

Comment:
Not your mistake but please could you remove the two following `print`s

Reply:
removed prints